api/payloads/responses.go

```diff
@@ -10,3 +10,27 @@ type HealthcheckResponse struct {
 type LoginResponse struct {
 	Token string `json:"token"`
 }
+
+type NewProjResponse struct {
+	ID string `json:"id"`
+}
+
+type CreateDeployKeyResponse struct {
+	DeployKey string `json:"deployKey"`
+}
+
+type AddOwnerResponse struct {
+	TeamKey string `json:"teamKey"`
```

Comment:
these addXResponse should all be addUserResponse, they also shouldnt return anything, just the 200 OK.